.github/workflows/shared.yml

```diff
@@ -45,4 +45,7 @@ jobs:
 
       - name: Run pytest
         run: uv run --frozen --no-sync pytest
-    continue-on-error: true
```

Comment:
This shouldn't be set, should it be?

Reply:
I don't think it's needed, but I figured out why it's here: so if other tests on the matrix fail, it will continue to run.